sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -73,11 +50,8 @@ export { OpenCensusTracer }
 export class OpenCensusTracerWrapper implements Tracer {
     constructor(tracer: TracerBase);
     bind<T>(_target: T, _span?: Span): T;
-    getBinaryFormat(): BinaryFormat;
-    getCurrentSpan(): Span | null;
-    getHttpTextFormat(): HttpTextFormat;
```

Comment:
What happened to the binary and http formats? I recall these were added in recently

Reply:
Binary format was removed here: https://github.com/open-telemetry/opentelemetry-js/pull/804

Looks like it was removed from the spec too from being underspecified: https://github.com/open-telemetry/opentelemetry-specification/pull/426


Http format was removed here: https://github.com/open-telemetry/opentelemetry-js/pull/816

Looks like it's because of some thinking around "propagators" changing in the spec: https://github.com/open-telemetry/opentelemetry-specification/pull/440